joomla/joomla-cms
# Add Tag filter

Pull Request to add tag filter option to article blog menu item
### Summary of Changes

Adds a new parameter to the menuitem for category article blogs. The necessary code for the actual filtering is already present in the model for quite some time.
### Testing Instructions

Create a new menu item for the com_content articles blog and select a tag in the menu item.
You should only get the articles with that selected tag.
### Documentation Changes Required

None
